README.md

```diff
@@ -500,6 +502,19 @@ client.OnDisconnected(fn func())
 client.OnError(onError func(sc *SubscriptionClient, err error) error)
 ```
 
+#### Custom HTTP Client
+
+Use `WithWebSocketOptions` to customize the HTTP client which is used by the subscription client.
+
+```go
+client.WithWebSocketOptions(WebsocketOptions{
+	HTTPClient: &http.Client{
+		Transport: http.DefaultTransport,
+		Timeout: time.Minute,
+	}
+})
+```
+
```

Comment:
So AFAIK this function already exists. This is just adding to docs, which is a welcome change.